packages/shared/src/logger/logger.ts

```diff
@@ -37,6 +37,7 @@ class Logger {
       basicAuth: GRAFANA_BASIC_AUTH,
       format: winston.format.json(),
       replaceTimestamp: true,
+      timeout: 5000,
```

Comment:
The default was 30s. I dialed it down a bit. 30s is extremely slow for the user app.

Reply:
Totally agree. I must have set it to a really high value during development and didn't add a comment reminding myself to make the default more reasonable.